src/stories/Library/search-result-item/search-result-item-skeleton.scss

```diff
@@ -0,0 +1,13 @@
+.search-result-item {
+  &.ssc {
+    grid-template-columns: min-content 1fr;
+  }
+  .ssc-square {
```

Comment:
This does not follow our BEM practices - partly intentionally because `skeleton-screen-css` does not use BEM.

This might be worth noting somewhere. It is something that would apply for all skeletons so noting it in each file does not make much sense.  One way to do so would be to explicitly disable `plugin/stylelint-bem-namics` for the file but on the other hand the current implementation does not fail liniting anyways. I wonder why that is 🤔 .